### PR TITLE
Enabling easier testing of GCP installations

### PR DESCRIPTION
**Pull request by mirons-google**
This adds support for two forseti installation types

1. By release version OR
1. By branch name

### CODE REVIEW

File: deployment-templates/py/forseti-instance.py [hunk 1]
```diff
@@ -75,20 +93,6 @@ def GenerateConfig(context):
         'properties': {
             'zone': context.properties['zone'],
             'machineType': (
```
Comment (carise) [inline deployment-templates/py/forseti-instance.py]: I got an error running this DM template. Looks like this dict property isn't quite correct?

- code: MANIFEST_EXPANSION_USER_ERROR
  message: |
    Manifest expansion encountered the following errors: Error compiling Python code: invalid syntax (<string>, line 96) Resource: forseti-instance.py Resource: config

File: deployment-templates/py/forseti-instance.py [hunk 2]
```diff
@@ -201,6 +219,19 @@ def GenerateConfig(context):
 
 (echo "0 * * * * $USER_HOME/run_forseti.sh") | crontab -
 """.format(
+read -d '' RUN_FORSETI << EOF
```
Comment (carise) [inline deployment-templates/py/forseti-instance.py]: This doesn't look right.

Comment (mirons-google) [inline deployment-templates/py/forseti-instance.py]: Sorry, this should be fixed now.

**Comment by mirons-google** [comment]
Apologies on creating this PR early. PTAL.

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request merged**